src/IntroPage/index.js

```diff
@@ -163,4 +164,4 @@ export class PureIntro extends React.Component<Props> {
   }
 }
 
-export default withUser(redirectsLoggedIn(PureIntro));
+export default selectUrlBooking(withUser(redirectsLoggedIn(PureIntro)));
```

Comment:
Have you tested this with frontend? I'm afraid it's not going to work, URL is parsed on import, not when the component is being rendered => if you access the `kiwi.com` or default MMB page and then click through app onto single booking, I expect original URL is parsed, not the one with `account/bookings/XXX`

Reply:
I was also concerned about this initially, but it seems to work well when integrated to frontend locally(although one can never be *too* sure with frontend until it's deployed to our staging branch ). I believe it might be due to lazy-loading modules, eg. mmb module is loaded once the browser route matches the specific mmb URL. [here](https://gitlab.skypicker.com/frontend/frontend/blob/master/src/mmb/browser/render.js#L31)